docs/snippets/angular/button-story-with-emojis.ts.mdx

```diff
@@ -1,33 +1,36 @@
 ```ts
 // Button.stories.ts
 
-import { Meta } from '@storybook/angular/types-6-0';
-
-import Button from './button.component';
+import { Button } from './button.component';
 
 export default {
-  title: 'Components/Button',
   component: Button,
-} as Meta;
+};
 
-export const Primary = () => ({
-  props: {
-    label: 'Button',
-    background: '#ff0',
-  },
-});
+export const Primary = {
```

Comment:
Alternatively you can write

```js
export const Primary = {
  args: { 
    label: 'Button',
    backgroundColor: '#ff0',
  },
};
```

And same for stories below

Reply:
@shilman I've left it as is as this snippet is used [here](https://storybook.js.org/docs/angular/writing-stories/introduction#how-to-write-stories), and at this point in the docs, we're not mentioning args. That comes in the next [section](https://storybook.js.org/docs/angular/writing-stories/introduction#using-args)

Reply:
👍